packages/ui/package.json

```diff
@@ -6,7 +6,8 @@
    "license": "MIT",
    "dependencies": {
       "@ifixit/app": "workspace:*",
-      "@ifixit/cart-sdk": "workspace:*"
+      "@ifixit/cart-sdk": "workspace:*",
+      "@ifixit/commerce-frontend": "workspace:*"
```

Comment:
What's the reason for adding commerce-frontend as a dep of the ui package?

Reply:
Gonna pull @andyg0808 in for this one. I wasn't sure if I needed to include it, but I remembered him saying I might need it (he helped me set this up). Do you not think it is needed? Easy change to remove it.

Reply:
If we didn't need to depend on it, I'm totally fine with not adding it as a dependency. I'd love to avoid it if we can. I think I thought we were gonna need files from it.

Reply:
In that case, I'm gonna go ahead and remove it. Thanks y'all!

Reply:
`@ifixit/commerce-frontend` is the package that contains the next.js app, it would be weird for a UI package to depend on the app package. If ui package needs some code that currently is in Next.js, then the best thing to do is to extract that piece of code into a separate package. Otherwise we'd start having circular dependencies between `@ifixit/commerce-frontend` and ui package.

Reply:
Ooh, ok. That makes sense. I don't see why we would need any Next.js code, so I think we should be good. Thanks @dhmacs !